src/utils/locale.ts

```diff
@@ -111,6 +113,10 @@ export const Languages = {
     locale: 'ku',
     dir: Direction.RTL,
   },
+  131: {
```

Comment:
Hi, I am trying to add another language and it is my first time working on localizing the Next.js app, I see in the `locale.ts` file, that the `Languages` object has numbers as a key for language. Is this some language code or identifier for specific language ?

Reply:
Assalamu alaikum, sorry for late response, I forgot about it. The way I managed to find language ID for Polish is the following: put `console.log(languageId)` in file src/utils/locale.ts in function "getLanguageDataById", then go to the app that you ran in a local environment, then pick any surah to read (translation mode, not reading mode), then go to settings, scroll down to translations and pick translation of Quran in your language (I hope it's available). Then your language ID should appear in developer tools of your browser as a log. You might need to refresh the page. Let me know if you need more help in sha Allah. I can check it for you, but I don't know which language you are trying to add to the website.

Reply:
Wa alaikum assalam, thanks for your response. It worked!

Reply:
Awesome! :slightly_smiling_face: